src/index.html

```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars--0 falling">
+      <div class="falling__star"></div>
+      <div class="falling__star"></div>
+      <div class="falling__star"></div>
+      <div class="falling__star"></div>
+      <div class="falling__star"></div>
+    </div>
```

Comment:
```suggestion
    <div class="stars stars--0">
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
    </div>
```
What does falling mean here?)

Reply:
Я хотела назвать звездопад :smile:

Reply:
Можешь оставить falling, но исправь по БЭМ по примеру, как я выше написала. stars--0 берется ниоткуда у тебя, блока stars нигде нет.